README.md

```diff
@@ -1,11 +1,11 @@
-![](./docs/images/karpenter-banner.png)
+![](docs/images/karpenter-banner.png)
 
 Karpenter is a node autoscaler built for Kubernetes and can run in any Kubernetes cluster anywhere.
 
 This is an early stage, experimental project built with ❤️ and is available as a **developer preview**. We're excited you are here - jump in, let us know what you think. We welcome contributions.
 ## Installation
 Follow the setup recommendations of your cloud provider.
-- [AWS](./docs/aws/README.md)
+- [AWS](docs/aws/README.md)
```

Comment:
Instead of README.md, can we just do /docs/aws? The readme will render.

Reply:
done